### PR TITLE
Flashlight suduko

### PR DESCRIPTION
**Pull request by ghost**
:cl:
tweak: you can burn your brains out with a flashlight if you're tired of life
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	H.adjustFireLoss(400)
```
Comment (swindly) [inline code/game/objects/items/devices/flashlight.dm]: return (FIRELOSS)

File: code/game/objects/items/devices/flashlight.dm [hunk 2]
```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (swindly) [inline code/game/objects/items/devices/flashlight.dm]: to their what?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/flashlight.dm [hunk 3]
```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] eyes and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```
Comment (RemieRichards) [inline code/game/objects/items/devices/flashlight.dm]: @Cyberboss I will not tell you again about [OBJECT] mr.

File: code/game/objects/items/devices/flashlight.dm [hunk 4]
```diff
@@ -36,6 +36,12 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/device/flashlight/suicide_act(mob/user)
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='suicide'>[user] is putting the [src] close to [user.p_their()] eyes and turning it on! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	return (FIRELOSS)
```
Comment (RemieRichards) [inline code/game/objects/items/devices/flashlight.dm]: it's tradition for ```suicide_act()```s... for some reason, it does no harm so either remove them all, or just leave it to be consistent.

File: code/game/objects/items/devices/flashlight.dm (hunk 3 above)
Comment (Cyberboss) [inline code/game/objects/items/devices/flashlight.dm]: I'm saying if he's going to use "the" it needs to be "\the" so it doesn't come out as "the the"

Comment (RemieRichards) [inline code/game/objects/items/devices/flashlight.dm]: yes, so the answer is just: "remove the", as having the \the macro there is redundant

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**